projects/rero/ng-core/src/public-api.ts

```diff
@@ -67,4 +69,4 @@ export * from './lib/utils/sort-by-keys';
 export * from './lib/utils/utils';
 export * from './lib/validator/time.validator';
 export * from './lib/validator/unique.validator';
-export * from './lib/error/error.component';
```

Comment:
Why removing this export, I think it is needed.

Reply:
done